src/commands/emulators-exec.ts

```diff
@@ -17,6 +18,13 @@ async function runScript(script: string): Promise<number> {
 
   const env: NodeJS.ProcessEnv = { ...process.env };
 
+  const databaseInstance = EmulatorRegistry.get(Emulators.DATABASE);
+  if (databaseInstance) {
+    const info = databaseInstance.getInfo();
+    const address = `${info.host}:${info.port}`;
+    env[DatabaseEmulator.DATABASE_EMULATOR_ENV] = address;
```

Comment:
We should add another log line to the controller like the one we have for Firestore:
https://github.com/firebase/firebase-tools/blob/master/src/emulator/controller.ts#L159